system-parachains/coretime/coretime-kusama/src/lib.rs

```diff
@@ -494,18 +494,36 @@ impl InstanceFilter<RuntimeCall> for ProxyType {
 	fn filter(&self, c: &RuntimeCall) -> bool {
 		match self {
 			ProxyType::Any => true,
-			ProxyType::NonTransfer => !matches!(
+			ProxyType::NonTransfer => matches!(
 				c,
-				RuntimeCall::Balances { .. } |
-				// `purchase`, `renew`, `transfer` and `purchase_credit` are pretty self explanatory.
-				RuntimeCall::Broker(pallet_broker::Call::purchase { .. }) |
-				RuntimeCall::Broker(pallet_broker::Call::renew { .. }) |
-				RuntimeCall::Broker(pallet_broker::Call::transfer { .. }) |
-				RuntimeCall::Broker(pallet_broker::Call::purchase_credit { .. }) |
-				// `pool` doesn't transfer, but it defines the account to be paid for contributions
-				RuntimeCall::Broker(pallet_broker::Call::pool { .. }) |
-				// `assign` is essentially a transfer of a region NFT
-				RuntimeCall::Broker(pallet_broker::Call::assign { .. })
+				RuntimeCall::System(_) |
+					RuntimeCall::ParachainSystem(_) |
+					RuntimeCall::Timestamp(_) |
+					RuntimeCall::CollatorSelection(_) |
+					RuntimeCall::Session(_) |
+					RuntimeCall::Utility(_) |
+					RuntimeCall::Multisig(_) |
+					RuntimeCall::Proxy(_) |
+					// We don't allow `purchase`, `renew`, `transfer`, `purchase_credit`,
+					// `pool` doesn't transfer, but it defines the account to be paid for contributions,
+					// `assign` is essentially a transfer of a region NFT.
+					RuntimeCall::Broker(pallet_broker::Call::configure { .. }) |
+					RuntimeCall::Broker(pallet_broker::Call::reserve { .. }) |
+					RuntimeCall::Broker(pallet_broker::Call::unreserve { .. }) |
+					RuntimeCall::Broker(pallet_broker::Call::set_lease { .. }) |
+					RuntimeCall::Broker(pallet_broker::Call::start_sales { .. }) |
+					RuntimeCall::Broker(pallet_broker::Call::partition { .. }) |
+					RuntimeCall::Broker(pallet_broker::Call::claim_revenue { .. }) |
+					RuntimeCall::Broker(pallet_broker::Call::drop_region { .. }) |
+					RuntimeCall::Broker(pallet_broker::Call::drop_contribution { .. }) |
+					RuntimeCall::Broker(pallet_broker::Call::drop_history { .. }) |
+					RuntimeCall::Broker(pallet_broker::Call::drop_renewal { .. }) |
```

Comment:
Besides this one, the  other calls should not be required.

Reply:
yes ok, `partition` could be added, but I kept it removed. done https://github.com/polkadot-fellows/runtimes/pull/646/commits/48b98377d8b5d87fb3f1a02e98acf56fc4f41433